crud/request.go

```diff
@@ -0,0 +1,109 @@
+package crud
+
+import (
+	"context"
+
+	"github.com/tarantool/go-tarantool"
+)
+
+type baseCrudRequest struct {
+	impl *tarantool.CallRequest
```

Comment:
It will simplify our life when `Call` became `Call17` by default.

```suggestion
	impl *tarantool.Call17Request
```